tests/repack-kernel/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: tests for the repack-kernel tool
+
+# ubuntu-core-initramfs seems to be only available on ubuntu-20.04
+systems: [ubuntu-20.04-64]
+
+execute: |
+    # install dependencies
+    apt update -yqq
```

Comment:
we use to install in prepare and then remove in restore, so we know the environment is clean after the test is executed.
In case of snapd, I would install it in spread.yaml, in the prepare section of the test suite, so it keeps installed for all the tests.